src/main/zapHomeFiles/hud/drawer.html

```diff
@@ -54,6 +54,11 @@
                         <th class="title hidden">
                             <span>{{ historyItemsFilteredMessage }}</span>
                         </th>
+                        <th class="title">
+                            <span>
+                                <button @click="clearHttpMessages()">Clear</button>
```

Comment:
The GUI should not have hard coded display strings in it (like "Clear") - those cannot be i18ned. Instead use something like `:name="$t('message.history_tool')"` - thse tags are defined in src/other/resources/UIMessages/UIMessages.properties

Reply:
The rest of the HTML uses these so you should be able to find plenty of examples, but if not just let us know.
Othert than that the PR seems to work well :)

Reply:
Done, thanks 😀